apps/web-giddh/src/app/shared/header/header.component.ts

```diff
@@ -1367,12 +1398,15 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
 
     public openExpiredPlanModel(template: TemplateRef<any>) { // show expired plan
```

Comment:
add return type

Reply:
@rohit-walkover Did you check if this function is existing or new?

Reply:
@rohit-walkover did you check if this function is existing or new?